modulemd/modulemd-component-rpm.c

```diff
@@ -424,6 +493,22 @@ modulemd_component_rpm_class_init (ModulemdComponentRpmClass *klass)
                          "The URI of the SCM repository.",
                          CR_DEFAULT_STRING,
                          G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS);
+  properties[PROP_BUILDROOT] = g_param_spec_boolean (
```

Comment:
Just a nitpick, but could you wrap the properties with `// clang-format off` and `// clang-format on` and get the alignment for all of the assignments lined up the same way?

Reply:
Sure! Any objections if I re-format the above properties to use the alignment I used for the new properties?

Reply:
Please do